core/dependencies/yaml-cpp

```diff
@@ -1 +1 @@
-Subproject commit 95bb6a18f9523b46499e9b870ef4b8e4d8068fbb
+Subproject commit a49337024219f7b3778cbcbfc3e6d26e84f9408f
```

Comment:
I had to do the following to get it working, since the yaml-cpp submodule's fork was updated:

```
git submodule sync
```